javascript/chronometer.js

```diff
@@ -1,26 +1,28 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => this.currentTime++, 1000);
   }
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime/60);
   }
   getSeconds() {
-    // ... your code goes here
```

Comment:
I don't think that `Math.floor()` is necessary here, the remainder will always be a whole number so I don't think there's need to use it here